src/index.ts

```diff
@@ -60,7 +61,8 @@ export const ReactMediaRecorder = ({
   screen = false,
   mediaRecorderOptions = null,
 }: ReactMediaRecorderProps) => {
-  const mediaRecorder = useRef<MediaRecorder | null>(null);
+  window.MediaRecorder = AudioRecorder
```

Comment:
May be overwrite only if mediate doesn’t exist like suggested in the audio recorder polyfill

+   <script>
+     if (!window.MediaRecorder) {
+       document.write(
+         decodeURI('%3Cscript defer src="/polyfill.js">%3C/script>')
+       )
+     }
+   </script>